app/knights_setup/knights_data.py

```diff
@@ -0,0 +1,86 @@
+KNIGHTS = {

```

Comment:
Rename module Instead of `knighs_setup` to `knight`, it could have files that are used for other purposes than the setup in the future.